GliaWidgets/SecureConversations/ChatTranscript/SecureConversations.TranscriptModel.swift

```diff
@@ -400,22 +399,8 @@ extension SecureConversations.TranscriptModel {
     }
 
     func linkTapped(_ url: URL) {
-        switch url.scheme?.lowercased() {
-        case "tel",
-            "mailto":
-            guard
-                environment.uiApplication.canOpenURL(url)
-            else { return }
-
-            environment.uiApplication.open(url)
-
-        case "http",
-            "https":
-            delegate?(.openLink(url))
-
-        default:
-            return
-        }
+        guard environment.uiApplication.canOpenURL(url) else { return }
```

Comment:
What if it cannot open? Nothing will happen? Perhaps it's not that bad of an experience, kind of like just pressing regular text.

Reply:
> What if it cannot open? Nothing will happen?

You are correct. Nothing will happen.
Do you think we need to show an error or/and send it to logger, in case when iOS can not open custom URL scheme, because of there is no app associated with this scheme?
Previously we just skipped handling of URLs with custom schemes at all.